website/project/views/node.py

```diff
@@ -586,20 +543,35 @@ def togglewatch_post(**kwargs):
         'watched': user.is_watching(node)
     }
 
+@must_be_logged_in
+@must_be_valid_project
+@must_not_be_registration
+@must_have_permission(ADMIN)
+def update_node(auth, node, **kwargs):
+    try:
+        return {
+            'updated_fields': {
+                key: getattr(node, key)
+                for key in
+                node.update(js_to_python(request.json))
```

Comment:
This seems dangerous; what if the client sends fields that aren't meant to be editable? It doesn't appear that `node.update` enforces any limitation on what can be updated.